Makefile.am

```diff
@@ -9,26 +9,40 @@ AM_DISTCHECK_CONFIGURE_FLAGS = \
 
 dist_systemdunit_DATA = \
 	eos-add-flatpak-gnome-repos.service \
-	eos-firstboot.service \
-	eos-extra-resize.service \
 	eos-enable-extra-upgrade.service \
 	eos-enable-zram.service \
+	eos-extra-resize.service \
+	eos-firewall-localonly.service \
+	eos-firstboot.service \
+	eos-fix-flatpak-branches.service \
 	eos-image-boot-dm-setup.service \
 	eos-live-boot-overlayfs-setup.service \
-	eos-fix-flatpak-branches.service \
 	$(NULL)
 
+# Network Manager dispatcher script for the firewall - scripts which
+# are a symlink to no-wait.d will be run without blocking/ordering.
+# install the script here and make the symlink in install-data-hook
+# below
+networkscriptdir = $(sysconfdir)/NetworkManager/dispatcher.d
+networkscriptnowaitdir = $(networkscriptdir)/no-wait.d
+dist_networkscriptnowait_DATA = eos-firewall-localonly-nm
```

Comment:
We want the script to be updatable via ostree, which will be the case if it lives in /etc and is not modified. We could host it somewhere off-path in /usr and place a symlink in /etc - I'd be happy to do that if it's preferred.